src/README.md

```diff
@@ -9,6 +9,7 @@ Orchard Core consists of two different targets:
 
 It’s important to note the differences between the framework and the CMS. Some developers who want to develop SaaS applications will only be interested in the modular framework. Others who want to build administrable websites will focus on the CMS and build modules to enhance their sites or the whole ecosystem.
 
+[![Orchard Core CN 中文讨论组](https://pub.idqqimg.com/wpa/images/group.png)](//shang.qq.com/wpa/qunwpa?idkey=48721591a71ee7586316604a7a4ee99d26fd977c6120370a06585085a5936f62)
```

Comment:
looks like it misses a "https:" in the front of "//shang.qq...."

Reply:
This link will evoke the QQ, or jump to the QR code interface

Reply:
The QQ group number is 877196442, welcome to join us 😎

Reply:
O. I see. It works.